pkg/controller/disruption/disruption_test.go

```diff
@@ -23,9 +23,8 @@ import (
 	"testing"
 	"time"
 
-	apps "k8s.io/api/apps/v1beta1"
+	apps "k8s.io/api/apps/v1"
```

Comment:
Honestly, I'd prefer being explicit in the code with these imports, iow. `appsv1` should be used here. This way it's explicit what you're doing in the code.

Reply:
will make a note for the future, was trying to keep the diff minimal for this PR